internal/store/graph_store_neo.go

```diff
@@ -208,3 +217,75 @@ func (neo *Neo) PutUser(user *model.User) error {
 
 	return nil
 }
+
+// GetUserSuggestion get user suggestions
+func (neo *Neo) GetUserSuggestion(user *model.User) (*model.Suggestion, error) {
+	logger := logrus.WithFields(logrus.Fields{
+		"logger":    "store/Neo.GetUserSuggestion",
+		"user.name": user.Name,
+	})
+
+	logger.Info("get user suggestion")
+
+	// keep start time for query metrics
+	startTime := time.Now()
+
+	// create template for query
+	neoGetUserSuggestionQuery, err := template.
+		New("neoGetUserSuggestionQuery").
+		Parse(neoGetTopStarredRepositories)
+	if err != nil {
+		return &model.Suggestion{}, errors.Wrap(err, "could not parse template")
+	}
+
+	// render query
+	query := &bytes.Buffer{}
+	type InputQuery struct {
+		Name      string
+		Timestamp int64
+	}
+	if err := neoGetUserSuggestionQuery.Execute(query, InputQuery{
+		Name:      user.Name,
+		Timestamp: startTime.Add(time.Hour * 24 * -7).Unix(),
+	}); err != nil {
+		return &model.Suggestion{}, errors.Wrap(err, "could not execute query")
+	}
+
+	logger.WithField("query", query).Debug("running query")
+
+	res := []struct {
+		NoOfFollowees int    `json:"noOfFollowees"`
+		Repository    string `json:"repository.name"`
+	}{}
+
+	// run query
+	cypherQuery := &neoism.CypherQuery{
+		Statement:  query.String(),
+		Parameters: map[string]interface{}{},
+		Result:     &res,
+	}
+	if err := neo.db.Cypher(cypherQuery); err != nil {
+		return &model.Suggestion{}, errors.Wrap(err, "could not run cypher query")
+	}
+
+	// log query time
+	logger.
+		WithField("execution_time", time.Now().Sub(startTime)).
+		Debug("query execution finished")
+
+	suggestions := make([]model.SuggestionItem, len(res))
+
+	for k, _ := range res {
+		suggestions[k] = model.SuggestionItem{
+			Type:   "repository",
+			Value:  res[k].Repository,
+			Reason: strconv.Itoa(res[k].NoOfFollowees) + " followers starred it",
```

Comment:
Just FYI, you could have done this as `fmt.Sprintf("%d followers starred it", res[k].NoOfFollowees)`